File: core/CreateTournament/Models/MatchResult.cs
using System.Collections.ObjectModel;
using System.ComponentModel.DataAnnotations.Schema;

namespace CreateTournament.Models
{
    public class MatchResult: DeleteableEntity
    {
        public int ScoreT1 { get; set; }
        public int ScoreT2 { get; set; }
        public int IdTeamWin { get; set; }
        public DateTime Finish { get; set; }
        public int MatchId { get; set; }
        [ForeignKey("MatchId")]
        public Match Match { get; set; }
        public Collection<PlayerStats> PlayerStats { get; set; }
    }
}

File: client/src/app/core/models/classes/MatchResult.ts
import { IMatchResult } from "../interfaces/IMatchResult";

export class MatchResult implements IMatchResult {
	id: number;
	scoreT1: number;
	scoreT2: number;
	idTeamWin: number;
	finish: Date;
	matchId: number;

	constructor() {
		this.scoreT1 = 0;
		this.scoreT2 = 0;
		this.idTeamWin = 0;
		this.matchId = 0;
		this.finish = new Date();
	}
}
